lib/msnodesqlv8/connection-pool.js

```diff
@@ -32,12 +33,20 @@ class ConnectionPool extends BaseConnectionPool {
         const key = p.substr(2, p.length - 3)
 
         switch (key) {
+          case 'driver':
+            // This allows windows users to choose thier driver
+            if (this.config.options.driver && !SUPPORTED_DRIVERS.includes(this.config.options.driver)) {
+              return reject('Unsupported driver: ' + this.config.options.driver)
+            }
+            return (this.config.options.driver ?? CONNECTION_DRIVER)
           case 'instance':
             return this.config.options.instanceName
           case 'trusted':
             return this.config.options.trustedConnection ? 'Yes' : 'No'
           case 'encrypt':
             return this.config.options.encrypt ? 'Yes' : 'No'
+          case 'columnEncryption':
+            return this.config.options.columnEncryption
```

Comment:
Does this not need to be converted from an actual boolean to `Yes` and `No` as per other options?

Reply:
I've updated the line to include a default
```
          case 'columnEncryption':
            return this.config.options.columnEncryption ?? 'Disabled'
```